awebox/mdl/dynamics.py

```diff
@@ -1060,37 +915,30 @@ def angular_velocity_inequality(options, variables, outputs, parameters, archite
     return outputs
 
 
-def tether_stress_inequality(options, variables, outputs, parameters, architecture):
+def tether_stress_inequality(options, variables_si, outputs, parameters, architecture):
     # system architecture (see zanon2013a)
     number_of_nodes = architecture.number_of_nodes
     kite_nodes = architecture.kite_nodes
     parent_map = architecture.parent_map
 
     # system (scaled) variables
```

Comment:
the comment says scaled variables, but the naming is `variables_si`. Which one is correct?

Reply:
si. :) thanks, inaccurate comment and unused variables removed.